### ISSUE TITLE
iOS 14 issue when we disable and enable video

### ISSUE THREAD

**Issue by iNima** [AgoraIO/Flutter-SDK#184]
**Describe the bug**
In iOS 14 when we call `await AgoraRtcEngine.disableVideo()` and then `await AgoraRtcEngine.enableVideo()` the video does not come back and it freezes.

**To Reproduce**
1. Start an agora live video
2. Tap a button to disable the video with `await AgoraRtcEngine.disableVideo()`
3. Tap another button to enable the video with `await AgoraRtcEngine.enableVideo()`
4. The video does not come back to continue streaming.

**Expected behavior**
The video should be enabled and continue streaming. 

**Smartphone (please complete the following information):**
 - Device: iPhoneX
 - Device OS: iOS 14
- Xcode: 11.5

**Additional context**
This issue only happens on iOS 14. On devices with iOS 13 or 12, it works correctly.
When this issue happens and we tap on another button to switch the camera `AgoraRtcEngine.switchCamera();` the video comes back again.

**Comment by iNima** [comment]
I fixed the issue which I think was specific to my project.

In my project, I stream live video via Agora but when the user wants to capture a photo I disable Agora video and switch to and use the Flutter Camera plugin.
When the camera finishes the capturing I re-enable the Agora video.
On iOS 13 and lower the Agora could start the video again without any issue but on iOS 14 it seems that it couldn't get the camera control back and use the camera.
I fixed this issue by disposing and nullifying the Flutter camera controller before enabling the Agora video.

Thanks and sorry because this issue was specific to my project.
